db/schema.rb

```diff
@@ -45,12 +45,28 @@
   end
 
```

Comment:
- [ ] Due to insufficient files changed I will leave a comment here. Kindly delete the seed data in your `seed.rb` file as it is  causing an error when migrating some files.

Reply:
👍🏽👌🏾

Reply:
- well resolved 👍🏽